spec/acceptance/api/cms_file_spec.rb

```diff
@@ -40,28 +40,25 @@
 
     context 'when the resource is a new record' do
       let(:resource) { FactoryBot.build(:cms_file, provider: provider, section: provider.sections.root) }
-      let(:root) { 'file' }
       let(:expected_attributes) { %w[section_id path downloadable url title content_type] }
 
-      json(:resource, skip_resource_save: true) do
+      json(:resource, skip_resource_save: true, skip_root_check: true) do
         it 'should have the correct attributes' do
           expect(subject.keys).to eq(expected_attributes)
         end
       end
     end
 
     context 'when requesting a single resource' do
-      let(:root) { 'file' }
-
-      json(:resource) do
+      json(:resource, skip_root_check: true) do
         it 'should have the correct attributes' do
           expect(subject.keys).to eq(expected_attributes)
         end
       end
     end
 
     context 'when requesting a collection' do
-      let(:root) { 'files' }
+      let(:root) { 'collection' }
 
       json(:collection)
     end
```

Comment:
I fail to understand rspecs, what's the assertion here?

Reply:
Just added a couple tests in 88e100e.